README.md

```diff
@@ -1,91 +1,23 @@
-# Mantle
+<p align="center">
+	<picture>
+		<source media="(prefers-color-scheme: dark)" srcset="https://raw.githubusercontent.com/ngrok-oss/mantle/refs/heads/readme-revamp/.github/mantle-dark.svg">
```

Comment:
Once approved, I will change all these URLs to `https://raw.githubusercontent.com/ngrok-oss/mantle/HEAD/` and not reference this branch.